src/capsule/CapsuleSigner.ts

```diff
@@ -22,24 +22,49 @@ const TAG = 'geth/CapsuleSigner'
 /**
  * Implements the signer interface using the CapsuleSignerModule
  */
+
+const USER_NOT_AUTHENTICATED_ERROR = 'user must be authenticated'
```

Comment:
It'd be great if these error constants were specified in the `UserManagementClient` to stay in-sync with the server.